dipy/reconst/dti.py

```diff
@@ -1151,6 +1158,28 @@ def predict(self, gtab, S0=1):
         return tensor_prediction(self.model_params[0:12], gtab, S0=S0)
 
 
+def tensor6_to_dtiparams(tensor, min_diffusivity=0):
```

Comment:
But it might be worth refactoring that function. For example, it currently does not take into account `min_diffusivitiy`